src/validation/ValidatorOptions.ts

```diff
@@ -3,6 +3,11 @@
  */
 export interface ValidatorOptions {
 
+    /**
+     * If set to true than validator will skip validation of all properties that are undefined in the validating object.
+     */
+    skipUndefinedProperties?: boolean;
+
     /**
      * If set to true than validator will skip validation of all properties that are missing in the validating object.
```

Comment:
Should you also update this documentation about `skipMissingProperties`?

Reply:
What do you mean? `skipMissingProperties` still checks `null` and `undefined` values.
Do you want to replace

> that are missing

with

> that are null or undefined

?
Or do you want to have a new option `skipNullProperties`?

Reply:
Yes replace missing documentation with `that are null or undefined` or something similar :)

Reply:
Done